cmd/goal/clerk.go

```diff
@@ -1262,25 +1230,73 @@ var simulateCmd = &cobra.Command{
 	Short: "Simulate a transaction or transaction group with algod's simulate REST endpoint",
 	Long:  `Simulate a transaction or transaction group with algod's simulate REST endpoint under various configurations.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		data, err := readFile(txFilename)
-		if err != nil {
-			reportErrorf(fileReadError, txFilename, err)
+		txProvided := cmd.Flags().Changed("txfile")
+		requestProvided := cmd.Flags().Changed("request")
+		if txProvided == requestProvided {
+			reportErrorf("exactly one of --txfile or --request must be provided")
+		}
+
+		requestOutProvided := cmd.Flags().Changed("request-out")
+		outProvided := cmd.Flags().Changed("outfile")
+		if requestOutProvided && outProvided {
+			reportErrorf("--request-out and --outfile are mutually exclusive")
```

Comment:
similarly, we should say 
```suggestion
			reportErrorf("--request-out and --result-out are mutually exclusive")
```
right?